README.md

```diff
@@ -50,6 +50,42 @@ NodeBB requires the following software to be installed:
 
 [Please refer to platform-specific installation documentation](https://docs.nodebb.org/installing/os)
 
+## Docker usage
+
+An example instance of NodeBB can also be rapidly started using Docker Compose provided in this repo.
+At least Docker Compose version 3.5 is required.
+
+### Choosing your database
+
+You will need to choose the type of database first. In this repo you will find three files
+- docker-compose.mongo.yml
+- docker-compose.postgres.yml
+- docker-compose.redis.yml
```

Comment:
I think now this can be moved to [profiles](https://docs.docker.com/compose/profiles/). Then it could all be in a single file that could be run with e.g. `docker-compose --profile mongo up` instead of having to manage two files separately.

Reply:
Can Compose services depends on services that are probably nonexistent?

Reply:
Hmm... I was thinking of overloading `db` instead, but it seems it's not really possible (which I guess makes sense since you can have >1 profile active), so perhaps the current implementation is the best way to do this.

Reply:
![image](https://user-images.githubusercontent.com/29133953/178157776-e41596b1-02ef-4a87-869a-68436e63b948.png)
that's the problem